### DISCUSSION TITLE
Got approved on the 25th but I still can't access benefits

### DISCUSSION THREAD

**Discussion by wisdomseller** [community/community#65128] [GitHub Education]
### Hi everyone,

I still don't have access to the benefits, I thought it took around 2 days or less, is this normal?

**Comment by nooma-42** [comment]
same here

**Comment by markvfrance** [comment]
same here, on the 25th

**Comment by advayc** [comment]
It takes 72 hours after the application to get your benefits. You'll get a email along with the pro badge on your github profile.
